### PR TITLE
How2 speech translation recipe

### PR DESCRIPTION
**Pull request by hirofumi0810**
- `st1`
- `asr1`

### CODE REVIEW

File: egs/how2/asr1/conf/tuning/decode_pytorch_transformer.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+batchsize: 0
```
Comment (sw005320) [inline egs/how2/asr1/conf/tuning/decode_pytorch_transformer.yaml]: It seems that the decoding configs of RNN and transformer are very different. If you don't test either of them, you can remove that config.

**Comment by hirofumi0810** [comment]
> Do you have some results? Can you add them as a `RESULTS.md`?

Let me do this in another PR. (due to some personal issues)

File: egs/how2/asr1/conf/tuning/decode_pytorch_transformer.yaml (hunk 1 above)
Comment (hirofumi0810) [reply egs/how2/asr1/conf/tuning/decode_pytorch_transformer.yaml]: I want to keep this as it is.

**Comment by sw005320** [comment]
Thanks. Please let me know when it is ready to merge.

**Comment by hirofumi0810** [comment]
This PR is ready to be merged.

File: egs/how2/st1/run.sh [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+fi
+
+feat_tr_dir=${dumpdir}/${train_set}/delta${do_delta}; mkdir -p ${feat_tr_dir}
+feat_dt_dir=${dumpdir}/${train_dev}/delta${do_delta}; mkdir -p ${feat_dt_dir}
+if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
+    ### Task dependent. You have to design training and dev sets by yourself.
+    ### But you can utilize Kaldi recipes in most cases
+    echo "stage 1: Feature Generation"
+
+    # The fbank features is generated as pre-processing; 40-dimensional fbanks with 3-dimensional pitch on each frame
```
Comment (sw005320) [inline egs/how2/st1/run.sh]: Why don't you include the feature extraction process?
I want `run.sh` to be self-consistent.

Comment (hirofumi0810) [reply egs/how2/st1/run.sh]: How2 corpus provides pre-processed fbank features (40-dim fbank+3-dim pitch) only, which means we have no chance to access to the original audio files.

Comment (sw005320) [reply egs/how2/st1/run.sh]: Thanks!
Now I got it.
Could you add such comments more explicitly in `run.sh`?
Then, it's perfect.

**Comment by sw005320** [comment]
Thanks a lot, @hirofumi0810!

**Pull request merged**